examples/acados/static_arm.py

```diff
@@ -110,18 +113,18 @@ def main():
         n_threads=6,
     )
 
+    opts_ipopt = SolverOptionsIPOPT()
+    opts_ipopt.set_solver('ma57')
+    opts_ipopt.dual_inf_tol = 1e-3
+    opts_ipopt.constr_viol_tol =  1e-3
+    opts_ipopt.compl_inf_tol = 1e-3
+    opts_ipopt.max_iter = 100
+    opts_ipopt.hessian_approximation = "exact"
```

Comment:
Same here, I they example, please be as much as by the book as possible. The goal of your new class is to provide easy accessors to things like `max_iter` so use them!